### PR TITLE
Redux in component

### PR DESCRIPTION
**Pull request by megha-n-bodke**
- Connect your app to the Redux store.
- Use useSelector to consume the state.
- Use Redux useDispatch.
- Implement add book and remove book functionality.
- Implement check status functionality.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -1,22 +1,28 @@
 const ADD_BOOK = 'books / books / ADD_BOOK';
 const REMOVE_BOOK = 'books / books / REMOVE_BOOK';
```
Comment (brytebee) [inline src/redux/books/books.js]: - [ ] **Required:** Kindly remove the white spaces in your action string. For example `const ADD_BOOK = 'books / books / ADD_BOOK';` should be `const ADD_BOOK = 'books/books/ADD_BOOK';`. This is to ensure it complies with the duck pattern convention.

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -1,14 +1,17 @@
 const categories = [];
 const CHECK_STATUS = 'categories/categories / CHECK_STATUS';
```
Comment (brytebee) [inline src/redux/categories/categories.js]: Same as above.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @megha-n-bodke & @OpondoG👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js [hunk 3]
```diff
@@ -1,22 +1,28 @@
-const ADD_BOOK = 'books / books / ADD_BOOK';
-const REMOVE_BOOK = 'books / books / REMOVE_BOOK';
-export const addBook = () => ({
-  type: ADD_BOOK,
-});
+const ADD_BOOK = 'books/books/ ADD_BOOK';
```
Comment (topeogunleye) [inline src/redux/books/books.js]: - [ ] [REQUIRED] As suggested by the previous reviewer, in order to follow the duck duck patterns. please remove the empty space here.  As an example 

❌ Wrong
const ADD_BOOK = 'books/books/ ADD_BOOK'; 

✔️ Right
const ADD_BOOK = 'books/books/ADD_BOOK';

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @megha-n-bodke, @OpondoG 👋 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* Good commit messages. :heavy_check_mark:
* No linter errors. :heavy_check_mark:
* Followed Github flow :heavy_check_mark:
* Professional README file. :heavy_check_mark:

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
- Nothing to mention.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions
-N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- NA

### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**